src/main/scala/sbtecr/AwsEcr.scala

```diff
@@ -13,19 +13,29 @@ private[sbtecr] object AwsEcr extends Aws {
 
   def domain(region: Region, accountId: String) = s"${accountId}.dkr.ecr.${region}.${region.getDomain}"
 
-  def createRepository(region: Region, repositoryName: String)(implicit logger: Logger): Unit = {
+  def createRepository(region: Region, repositoryName: String, repositoryPolicyTextOption : Option[String])(implicit logger: Logger): Unit = {
```

Comment:
I'd suggest changing the argument declaration as such: `repositoryPolicyText: Option[String] = None` (the "option" part comes from the type itself + the extra space).